e2e/testcases/helm_sync_test.go

```diff
@@ -110,9 +111,11 @@ func TestHelmNamespaceRepo(t *testing.T) {
 		nt.T.Error(err)
 	}
 	// Change the RepoSync to sync from the original git source to make test works in the shared test environment.
-	rs.Spec.SourceType = string(v1beta1.GitSource)
-	nt.RootRepos[configsync.RootSyncName].Add(nomostest.StructuredNSPath(repoSyncNN.Namespace, repoSyncNN.Name), rs)
-	nt.RootRepos[configsync.RootSyncName].CommitAndPush("Update RepoSync to sync from the original git repository")
+	nt.T.Cleanup(func() {
```

Comment:
The cleanup should be declared earlier in the test before a failure might occur. In this case it should probably be done when the `SourceType` is changed to use helm

Reply:
Sounds good, done.